api/config/routes.rb

```diff
@@ -91,14 +91,7 @@
     resources :states, only: [:index, :show]
 
     resources :taxonomies do
-      member do
-        get :jstree
-      end
-      resources :taxons do
-        member do
-          get :jstree
-        end
-      end
+      resources :taxons
```

Comment:
:-1: since this breaks the existing API. Anyone who has an iOS or Android app using these endpoints are going to get stuck on previous versions of Solidus.